components/style/properties/gecko.mako.rs

```diff
@@ -3784,13 +3784,20 @@ fn static_assert() {
         }
 
         self.clear_overflow_sides_if_string();
-        self.gecko.mTextOverflow.mLogicalDirections = v.second.is_none();
```

Comment:
Okay, feel free to land as is then (w/followup filed), no need to block stylo on pure servo changes.